sqldelight-studio-plugin/src/main/resources/META-INF/services/com.android.ide.common.blame.parser.PatternAwareOutputParser

```diff
@@ -0,0 +1 @@
+com.squareup.sqldelight.intellij.parsing.OutputParser
```

Comment:
i actually tried that first, but i think autoservice doesnt play well with kotlin, the implementation class it ended up with here was `PatternAwareOutputParser` (the superclass)